unified-codes/apps/web/src/components/templates/Header.tsx

```diff
@@ -51,7 +51,7 @@ export type Header = React.FunctionComponent<HeaderProps>;
 
 const _Header: Header = ({ classes }) => {
   return (
-    <AppBar position="static" className={classes.root}>
+    <AppBar position="fixed" className={classes.root}>
       <Grid item className={classes.title1}>
```

Comment:
I'm thinking if `Header` knows it needs `classes.title1`, would we lose anything by directly importing the class in the `Header` component file?